rkt/api_service.go

```diff
@@ -289,26 +283,27 @@ func getNetworks(p *pod) []*v1alpha.Network {
 func getBasicPod(p *pod) (*v1alpha.Pod, *schema.PodManifest, error) {
```

Comment:
> With this change, if you pass an Id into the api it will always echo it back, yes? 

@euank Nope. when getBasicPod is called, the pod is guaranteed to be there.